external-resolver/src/main/scala/org/genivi/sota/resolver/packages/PackageDirectives.scala

```diff
@@ -50,27 +49,13 @@ class PackageDirectives(namespaceExtractor: Directive1[Namespace])
     }
 
   def getPackageFilters(ns: Namespace, id: PackageId): Route =
-    completeOrRecoverWith(db.run(
-      PackageFilterRepository.listFiltersForPackage(ns, id))) {
-        Errors.onMissingPackage
-      }
+    complete(db.run(PackageFilterRepository.listFiltersForPackage(ns, id)))
 
   def addPackageFilter(ns: Namespace, id: PackageId, fname: String Refined Filter.ValidName): Route =
-    completeOrRecoverWith(db.run(
-      PackageFilterRepository.addPackageFilter(PackageFilter(ns, id.name, id.version, fname)))) {
-        Errors.onMissingPackage orElse Errors.onMissingFilter orElse { case err => throw(err) }
-      }
+    complete(db.run(PackageFilterRepository.addPackageFilter(PackageFilter(ns, id.name, id.version, fname))))
 
   def deletePackageFilter(ns: Namespace, id: PackageId, fname: String Refined Filter.ValidName): Route =
-    completeOrRecoverWith(db.run(
-      PackageFilterRepository.deletePackageFilter(PackageFilter(ns, id.name, id.version, fname)))) {
-        case PackageFilterRepository.MissingPackageFilterException =>
-          complete(StatusCodes.NotFound ->
-            ErrorRepresentation( ErrorCode("filter_not_found"),
-              s"No filter for namespace '$ns' with the name '${fname.get}'" +
-              s"defined for package ${id.name}-${id.version}"))
```

Comment:
It seems that we will not only print "No filter with defined for package" which seems less helpful. Am I right that we will have a less informative error message? Does that matter?